docs/en/writing-running-appium/ios-xctest-mobile-gestures.md

```diff
@@ -30,7 +30,7 @@ coordinates and duration.
 ```java
 // Java
 JavascriptExecutor js = (JavascriptExecutor) driver;
-Map<String, Object> params = new HashMap<>();
+Map<String, Object> scrollObject = new HashMap<>();
 scrollObject.put("direction", "down");
 scrollObject.put("element", ((RemoteWebElement) element).getId());
 js.executeScript("mobile: swipe", params);
```

Comment:
If the name is changed in line 33, it needs to be changed here, too. `params` no longer exists.